### PR TITLE
Added ability to decide if the annotation column should be filled automaticly and fixed wrong type determination when importing a dmn with annotation column

### PR DESCRIPTION
**Pull request by eric-hess**
While using the plugin I noticed two things.

1. If I export a DMN without annotation column and want to import it again, the last output column is reused as annotation column by default.
2. if I export a DMN with an annotation column and want to import it again, the types of the output columns are no longer transferred correctly to DMN.

__Steps to Reproduce__

1. 
- Create a new DMN with some sample data and without annoations
- Bildschirmfoto 2022-07-29 um 11 45 58
- Export this DMN
- Import this DMN again
- Bildschirmfoto 2022-07-29 um 11 46 22

2. 
- Create a new DMN with some sample data and with annotations
- Bildschirmfoto 2022-07-29 um 11 46 31
- Export this DMN
- Import this DMN again
- Bildschirmfoto 2022-07-29 um 11 46 51

__Changes__
- Test DMN without annotations
- Bildschirmfoto 2022-07-29 um 12 23 02
- Added new option to import dialog to decide if the imported excel contains a annotation column
- Bildschirmfoto 2022-07-29 um 12 20 48
- After import the annotation column is not filled
- Bildschirmfoto 2022-07-29 um 12 22 19
- Test DMN with annotations
- Bildschirmfoto 2022-07-29 um 12 23 47
- Select the new option to decide if the imported excel contains a annotation column
- Bildschirmfoto 2022-07-29 um 12 24 02
- After import the annotation column is filled and the correct output types are set
- Bildschirmfoto 2022-07-29 um 12 24 12

### CODE REVIEW

File: converter/excelHandler.js [hunk 1]
```diff
@@ -32,14 +32,18 @@ export const parseDmnContent = ({
...
+      hasAnnotationColumn
     } = sheetOptions;
 
     const rawInputData = header.slice(0, header.length - amountOutputs);
     const rawOutputData = header.slice(header.length - amountOutputs);
     const safeRuleRows = validateRows(sheet.data.slice(1));
+
     const typeRefs = getTypeRefs(safeRuleRows[0]);
 
+    console.log(typeRefs);
```
Comment (pinussilvestrus) [inline converter/excelHandler.js:45]:
```suggestion
```

Let's remove this one so we do not accidentally log stuff in production.

File: test/converter/excelHandlerSpec.js [hunk 2]
```diff
@@ -189,10 +280,8 @@ describe('excelHandler', () => {
       expect(dmnContent.rules[0].outputEntries.length).to.eql(expectedRules[0].outputEntries.length);
       expect(dmnContent.rules[0].outputEntries[0].text).to.eql(expectedRules[0].outputEntries[0].text);
     });
-
   });
 
-
```
Comment (pinussilvestrus) [inline test/converter/excelHandlerSpec.js:195]: Let's do not remove this line, as the tests always contain two spaces between two `describe` and `it`.

**Review by pinussilvestrus — CHANGES_REQUESTED** [review]
Really cool, thank you very much for this contribution 👏 🚀 

One thing before we can merge this: Can you please update the commit two a single one, following the existing commit message guideline? So something like

```text
fix: add option to mark annotation columns
```

This way we keep the commit history clean ❤️

File: converter/excelHandler.js (hunk 1 above)
Comment (eric-hess) [reply converter/excelHandler.js]: sure, my mistake 
[649c3e8](https://github.com/pinussilvestrus/camunda-modeler-excel-import-plugin/pull/109/commits/649c3e8730b4171d92714db46d9fb463e868b061)

File: test/converter/excelHandlerSpec.js (hunk 2 above)
Comment (eric-hess) [reply test/converter/excelHandlerSpec.js:195]: [649c3e8](https://github.com/pinussilvestrus/camunda-modeler-excel-import-plugin/pull/109/commits/649c3e8730b4171d92714db46d9fb463e868b061)

File: dist/client.js [hunk 3]
```diff
@@ -1028,12 +1037,14 @@ const parseDmnContent = ({
...
       tableName,
-      hitPolicy
+      hitPolicy,
+      hasAnnotationColumn
     } = sheetOptions;
     const rawInputData = header.slice(0, header.length - amountOutputs);
     const rawOutputData = header.slice(header.length - amountOutputs);
     const safeRuleRows = validateRows(sheet.data.slice(1));
     const typeRefs = getTypeRefs(safeRuleRows[0]);
+    console.log(typeRefs);
```
Comment (pinussilvestrus) [inline dist/client.js:1047]: I think you have to run `npm run build` again and push the changes.

File: test/converter/excelHandlerSpec.js [hunk 4]
```diff
@@ -189,10 +280,9 @@ describe('excelHandler', () => {
...
+
```
Comment (pinussilvestrus) [inline test/converter/excelHandlerSpec.js:285]: CI is failing because of a linting error. You can fix it by using `npm run lint -- --fix`.

**Review by pinussilvestrus — APPROVED** [review]
Thank you very much 👏

**Pull request merged**